source/AlertLabel.h

```diff
@@ -0,0 +1,50 @@
+/* AlertLabel.h
+Copyright (c) 2022 by Daniel Yoon
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef ALERT_LABEL_H_
+#define ALERT_LABEL_H_
+
+#include "Color.h"
+#include "Point.h"
+
+#include <memory>
+#include <string>
+
+class Projectile;
+class StellarObject;
+class System;
+class Ship;
+
+
+
+class AlertLabel {
+public:
+	AlertLabel(Point &position, const Projectile& projectile, std::shared_ptr<Ship> flagship, double zoom);
+
+	void Draw() const;
+
+
+private:
+	Point position;
+	double radius;
+	std::string bigText;
```

Comment:
those seem to be constant when they are set, so add them to the namespace in the cpp?
same for the colors